docs/troubleshooting.md

```diff
@@ -16,6 +18,22 @@ Potential solution: re-encode with a Windows supported audio sample rate
 
 ## "No compatible source was found" on IE11 Win 7
 
-videojs-http-streaming does not support Flash HLS playback (like the videojs-contrib-hls plugin did)
+videojs-http-streaming does not support Flash HLS playback (like the videojs-contrib-hls plugin does)
 
-Solution: please use the FlasHLS source handler https://github.com/brightcove/videojs-flashls-source-handler
+Solution: include the FlasHLS source handler https://github.com/brightcove/videojs-flashls-source-handler#usage
+
+## CORS: No Access-Control-Allow-Origin header
+
+If you see an error along the lines of
+
+```
+XMLHttpRequest cannot load ... No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin ... is therefore not allowed access.
+```
+
+you need to properly configure CORS on your server: https://github.com/videojs/http-streaming#hosting-considerations
+
+## Desktop Safari/iOS Safari/Android exhibit different behavior from other browsers
```

Comment:
I think most users will see a stream work on desktop Chrome and not on mobile Chrome or Edge. I think we should mention those browsers specifically.